Assignment2/Q1/src/NodeBinaryTree.java

```diff
@@ -0,0 +1,93 @@
+import javax.xml.soap.Node;
+
+/**
+ * Created by Alice Colt.
+ *
+ * Implementation of Node for BinaryTree with constructor, getters, setters and compareTo
+ *
+ */
+public class NodeBinaryTree
+{
+    // Value of node
```

Comment:
Comments like this aren't helpful - they tell you nothing that isn't already quite obvious.